lib/primer/base_service.rb

```diff
@@ -1,13 +1,22 @@
 module Primer
   class BaseService
     class << self
-      attr_reader :running_command
-      def set_running_command command
+      attr_reader :running_command, :port, :repo_name
+
+      def set_port(port)
+        @port = port
+      end
+
+      def set_repo_name(repo_name)
+        @repo_name = repo_name
+      end
```

Comment:
I don't see this used anywhere.

Reply:
@nick-desteffen It appears we use it here:

https://github.com/watermelonexpress/benchprep-v2/pull/10745/files#diff-17b0bf160be262cf71f54b2c48d1469677b6c24817ab69370fa7d60b6091b68aR10

Reply:
I think despite that we may be able to remove it entirely and just create a `getter`

https://github.com/watermelonexpress/benchprep-v2/pull/10745#discussion_r657088820